website/www/site/content/en/documentation/transforms/python/elementwise/pardo.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Partition"
```

Comment:
Another copy of "Partiton." (There may be others, we should verify we haven't lost content in the move.)